exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/CreateDatabaseTest.kt

```diff
@@ -22,6 +24,53 @@ class CreateDatabaseTest : DatabaseTestsBase() {
         }
     }
 
+    @Test
+    fun testListDatabasesOracle() {
+        withDb(TestDB.ORACLE) {
+            assertFailsWith<IllegalStateException> {
+                SchemaUtils.listDatabases()
+            }
+        }
+    }
+
+    @Test
+    fun testListDatabasesPostgres() {
+        withDb(TestDB.POSTGRESQL) {
```

Comment:
Could TestDB.POSTGRESQLNG be included here? It isn't covered by any of the other new tests since it's excluded from `testListDatabases()` below.

Reply:
I'm not sure it's possible: we're using the `assume` statement, and it will exit a test after the ignored `withDb`

Reply:
Done!